Bifrost.h

```diff
@@ -89,8 +91,24 @@ Bifrost::Bifrost(const std::string& img) : fInside(false)
     const std::string mount = fDir+":/bifrost";
 
     std::vector<const char*> args = {"singularity", "run", "--cleanenv",
-                                     "-B", mount.c_str(),
```

Comment:
What happened to `-B mount.c_str()`? That is needed for the pipes

Reply:
It's still there! See line 94 of the changed version.

Reply:
ugh, yes, I can't read diffs :(